contracts/interfaces/IJBToken.sol

```diff
@@ -4,7 +4,7 @@ pragma solidity 0.8.6;
 interface IJBToken {
   function totalSupply(uint256 _projectId) external view returns (uint256);
 
-  function balanceOf(uint256 _projectId, address _account) external view returns (uint256);
+  function balanceOf(address _account, uint256 _projectId) external view returns (uint256);
```

Comment:
doesn't JBToken balanceOf also need to updated?

Reply:
yaaaaa